handlers/controller.js

```diff
@@ -59,8 +59,8 @@ let getProduct = (req, res) => {
 
             let newProduct = new models.Product ({
                 barcode: barcode,
-                name: bclRes.data.product_name,
-                category: bclRes.data.category,
+                name: bclRes.data.name,
+                category: bclRes.data.category.split('>').map(Function.prototype.call, String.prototype.trim),
```

Comment:
Please make the following change:
Take the lowest level category (the last index in the array?) as the primary category for the product (This will be an array, but will only have one element initially and max 2). 
The rest should be secondary categories. (This will go into a new array). 

Thus we will need two fields, instead of just the one called category. 

A product will be able to have upto two primary categories, but the second one will have to be manually entered by either us or the users.

Reply:
Revise as per our discussion instead of this.